.gitignore

```diff
@@ -3,3 +3,4 @@
 *.rs.rustfmt
 target/
 Cargo.lock
+.idea/
```

Comment:
I'm guessing this is the IDE keep doing this, but just a hint in case it was intentional: It's recommended to keep user-specific ignore rules under `~/.gitignore` (or whatever ignore file you set in your `~/.gitconfig`) and not under the projects. Have you tried that?